build-system/tasks/performance/cache-documents.js

```diff
@@ -14,11 +14,26 @@
  * limitations under the License.
  */
 
-const {CONTROL, downloadToDisk, EXPERIMENT} = require('./helpers');
+const {
+  CONTROL,
+  copyImageToCache,
+  downloadToDisk,
+  EXPERIMENT,
+} = require('./helpers');
 const {startServer, stopServer} = require('../serve');
 const HOST = 'localhost';
 const PORT = 8000;
 
+// To be updated with local images in the documents in config.json
+const imageUrls = [
```

Comment:
Can we move these images to be derived from the documents requested?

Reply:
Yes, we can do something similar to how we cache the script files.

Reply:
Awesome, that would be much more resilient in the long term.